packages/lit-dev-content/site/tutorials/tutorials.njk

```diff
@@ -0,0 +1,6 @@
+---
+layout: false
+permalink: /tutorials/tutorials.json
+eleventyExcludeFromCollections: true
+---
+{ "tutorials": {{ tutorialsJson | safe }} }
```

Comment:
This lets you not have to modify `tutorials.11tydata.js`.
```suggestion
{ "tutorials": {{ tutorials | dump | safe }} }
```

p.s. What fumes are you huffing? I could use some inspiration like this. 😛